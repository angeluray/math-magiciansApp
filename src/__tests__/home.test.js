import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../pages/home';

describe('Check home function', () => {
  it('Home function renders the content', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
