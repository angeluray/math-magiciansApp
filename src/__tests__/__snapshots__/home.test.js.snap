// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check home function Home function renders the content 1`] = `
<div
  className="homeContainer"
>
  <section
    className="homeSection"
  >
    <h2>
      Welcome to Math Magicians!
    </h2>
    <p>
      The repetition of calculations reduces interest in mathematics. Students tend to get bored when classes are based on the monotonous resolution of sheets and sheets full of exercises.
    </p>
    <p>
      When the objective is to develop general mathematical reasoning abilities or research of numerical patterns, the calculator is necessary to avoid spending minutes solving long operations. We do it different with the Magic Calculator as the protagonist which will enchanted you to make all your Math operations here once you try it.
    </p>
  </section>
</div>
`;
